privaterelay/templates/profile.html

```diff
@@ -146,6 +146,10 @@
   </div>
 
   {% include "includes/domain-alias-dashboard.html" %}
+  {% is_dismissed "tips" as tips_dismissal%}
+  {% if user_profile.subdomain and not tips_dismissal and domain_addresses|length == 0 %}
+    {% include "includes/tips.html" %}
+  {% endif %}
```

Comment:
Yes, but indeed I wanted to cross that bridge when we get there, especially since the React-based website might be live before we get to implementing additional tips.

Reply:
Understood!